Makefile

```diff
@@ -321,8 +321,11 @@ go-lint: ## to lint go code
 	-v $(PWD):/app -w /app \
 	-e GOCACHE="/tmp" \
 	-e GOLANGCI_LINT_CACHE="/tmp" \
-	${DOCKER_IMAGE_GOLINT} \
-	golangci-lint run ./... || ${FAIL}
+	${DOCKER_IMAGE_GO} \
+	/bin/sh -c \
+	"cd /app && \
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLINT_VERSION) && \
```

Comment:
Why not just use the docker golangci lint image vs install the command?

Reply:
Because latest golang-ci image does not support golang 1.21.4 😞